components/eamxx/src/share/io/scorpio_output.cpp

```diff
@@ -359,10 +369,11 @@ set_grid (const std::shared_ptr<const AbstractGrid>& grid)
       (grid->get_global_max_dof_gid()-grid->get_global_min_dof_gid()+1)==grid->get_num_global_dofs(),
       "Error! In order for IO to work, the grid must (globally) have dof gids in interval [gid_0,gid_0+num_global_dofs).\n");
 
-  EKAT_REQUIRE_MSG(m_comm.size()<=grid->get_num_global_dofs(),
-      "Error! PIO interface requires the size of the IO MPI group to be\n"
-      "       no greater than the global number of columns.\n"
-      "       Consider decreasing the size of IO MPI group.\n");
+//ASD  IS THIS STILL TRUE vv
```

Comment:
We should verify this and, if no longer true, remove these lines.

Reply:
I think intuitively we can see that in regional output we will violate this.  The IO grid will be a subset of the main grid and if the regional output is small enough, say just a handful of ARM sites we can expect the number of mpi ranks to exceed the number of output points.  We may want to do something like have the `m_comm` just be a sub-comm of the main the comm group with just those ranks that are involved in writing the output.  Then we can keep this check.